app/_assets/javascripts/app.js

```diff
@@ -683,3 +683,18 @@ $(".closebanner").on("click", function () {
   $(".navbar-v2").addClass("closed");
   localStorage.setItem("closebanner-hackathon", "closebanner");
 });
+
+
+// Tooltips for badges
+jQuery(function () {
+    $('.badge.enterprise')
+      .append( '<div class="tooltip"><span class="tooltiptext">Available with Enterprise subscription</span></div>' );
+    $('.badge.plus')
+      .append( '<div class="tooltip"><span class="tooltiptext">Available with Plus subscription (Konnect Cloud)</span></div>' );
+    $('.badge.free')
+      .append( '<div class="tooltip"><span class="tooltiptext">Available in Free mode (not open-source)</span></div>' );
+    $('.badge.oss')
+      .append( '<div class="tooltip"><span class="tooltiptext" >Available in open-source only</span></div>' );
```

Comment:
This one IMO could be changed a tad. To me it seems slightly ambiguous and could be mistakenly parsed as:

> This feature is _only_ available in open-source

I think it's better without it "only":

> Available in open-source

Or also some variation of the following:

> Available in Kong OSS
> Available in Kong open-source

Reply:
Actually, `OSS ONLY` means exactly that - this is for features that are _only_ available in open-source. This is a very rarely used badge (right now, I think it's only on [this topic](https://deploy-preview-3523--kongdocs.netlify.app/gateway/2.7.x/plan-and-deploy/performance-testing-framework/)).

Reply:
^ relevant to the above but applicable pretty much anywhere: I know you probably feel like you type the work `Kong` 9 million times per day, but I think `Kong OSS` or `Kong open-source` is more readable than `open-source`. Putting `Kong` in front makes it much faster for my brain to parse out "this is a specific noun/thing/product they're referring to" (just like how you've capitalized the `Free` in `Free mode` above, making it clear that `Free` refers to a specific concept and not the generic meaning of `free`).

Reply:
Still added "Kong" to the name to hopefully clear up _some_ confusion. Hope it helps a little.